.golangci.yml

```diff
@@ -51,14 +50,6 @@ linters-settings:
     excludes:
       - G101 # Look for hard coded credentials
 
-  gofmt:
-    # Simplify code: gofmt -s
-    simplify: true
-    # Define indentation
-    rewrite-rules:
-      - pattern: "interface{}"
-        replacement: "any"
-
```

Comment:
This was 👍 from me. Since we're using `gofumpt` for auto-formatting (see `autofix` workflow), the `gofmt` is at odds with it.